fields/fields.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2022 Tigris Data, Inc.
```

Comment:
commenting it here so that we remember, we need to have Read and Update builder here, and then the usage would be fields.Read and fields.Update.